crates/db_schema/src/utils.rs

```diff
@@ -335,7 +336,12 @@ fn establish_connection(config: &str) -> BoxFuture<ConnectionResult<AsyncPgConne
         error!("Database connection failed: {e}");
       }
     });
-    AsyncPgConnection::try_from(client).await
+    let mut conn = AsyncPgConnection::try_from(client).await?;
+    conn
+      .batch_execute("SET geqo_threshold=12;SET fro_collapse_limit=11;SET join_collapse_limit=11;")
+      .await
+      .map_err(ConnectionError::CouldntSetupConfiguration)?;
+    Ok(conn)
```

Comment:
Please add some comments what these settings do and why you picked those values. I also think this could cause problems if an admin tries to optimize their db, but changes get silently overwritten by Lemmy.

Reply:
> I also think this could cause problems if an admin tries to optimize their db, but changes get silently overwritten by Lemmy.

I will eventually make queries not exceed the new collapse limits. Higher values set by admins will make no difference in that case.

Reply:
Added comment